src/AppBundle/Entity/LegislativeCandidate.php

```diff
@@ -14,34 +14,214 @@
 class LegislativeCandidate
```

Comment:
Well no in fact. Because we don't have full addresses. Admin will manually fill the coordinates from the backend area.